src/mono/sample/wasm/blazor-frame/blazor.csproj

```diff
@@ -11,8 +11,8 @@
 
   <ItemGroup>
     <!-- TODO un-pin this when it's possible -->
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="9.0.0-preview.1.24081.5" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="9.0.0-preview.1.24081.5" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="9.0.0-preview.3.24172.13" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="9.0.0-preview.3.24172.13" PrivateAssets="all" />
```

Comment:
Do we have an issue that tracks removing these hardcodes?

Reply:
> Do we have an issue that tracks removing these hardcodes?

Might want to ask that @radekdoulik @maraf @pavelsavara who were involved in the PR that introduced it; looks like this has been causing extra work when bumping the SDK since 8.0 RC1.